agent/recorder.go

```diff
@@ -168,7 +169,7 @@ func (r *SpanRecorder) sendSpans() (error, bool) {
 
 		var testSpans int64
 		for _, span := range spans {
-			if isTestSpan(span) {
+			if isTestSpan(span["tags"].(opentracing.Tags)) {
```

Comment:
Unrelated change?

Reply:
yes but important one, previously was not counting the test not being sent, and the output message was the "success" one, and not the "error/check the logs" one. I found it while testing the timeout fix.

Reply:
That's a different PR then

Reply:
reverted.